source/_docs/tools/quick-bar.markdown

```diff
@@ -59,6 +60,12 @@ Type | Available |
 | Reload | All currently-supported "Reload {domain}" services.<br />*(E.g., "Reload Scripts")* |
 | Server | Restart/Stop |
 
+## My links
+
+*Hotkey: `m`*
+
+Create [`my`](/integrations/my) links from any supported page in the user interface, when invoked on a supported page it will open a new tab which will allow you to share the link in different formats.
```

Comment:
```suggestion
Create [`my`](/integrations/my) links from any supported page in the user interface, when invoked on a supported page it will open a new tab that will allow you to share the link in different formats.
```